pkg/cidata/cidata.go

```diff
@@ -177,7 +177,7 @@ func GenerateISO9660(instDir, name string, y *limayaml.LimaYAML, udpDNSLocalPort
 		return errors.New("no SSH key was found, run `ssh-keygen`")
 	}
 	for _, f := range pubKeys {
-		args.SSHPubKeys = append(args.SSHPubKeys, f.Content)
+		args.SSHPubKeys = append(args.SSHPubKeys, strconv.Quote(f.Content))
```

Comment:
This won't work because the string will have two quotes in `user-data`

For example, the key `ssh-key "me@email.com"` will produces the following:
```
ssh-authorized-keys:
  - ""ssh-key \"me@email.com\"""
```
Instead, you can change template [user-data](https://github.com/lima-vm/lima/blob/master/pkg/cidata/cidata.TEMPLATE.d/user-data#L34-L37) like this:
```
    {{- range $val := .SSHPubKeys}}
      - |
        {{$val}}
    {{- end}}
```

Reply:
Sorry, this PR was stupid. Thanks for the suggestions. Created another at https://github.com/lima-vm/lima/pull/2194